css/style.css

```diff
@@ -0,0 +1,152 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 60%;
+}
+
+body{
+    font-family:'Times New Roman', Times, serif;
+    background-color: #2b2d42;
+}
+
+a{
+    text-decoration: none;
+    font-weight: 400;
+    /* 1.6*10 = 16p */
+    font-size: 2rem;
+    color: #8d99ae;
+
+}
+h2,p{
+    margin: 1.6rem 0;
+    color: #2b2d42
+}
+
+h1{
+    font-weight: 800;
+    font-size: 3.6rem;
+    color: #8d99ae;
+}
+
+h2{
+    font-weight: 600;
+    font-size: 2.4rem;
+    color: #2b2d42;
+}
+
+p{
+    font-weight: 400;
+    font-size: 0.8rem;
+}
+
+img{
+    width: 100%;
+    height: auto;
+}
+
+.flex-container{
+    display: flex;
+    justify-content: space-between;
+    width: 80%;
+    align-items: center;
+    margin: auto;
+}
+
+
+nav{
+    display: flex;
+    justify-content: space-between;
+}
+
+.logo h1{
```

Comment:
targeting a name is better, insted of using h1 you can use logo-title or logoTitle so naming every element with semantic name is the first goal always.